go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535 // indirect
 	github.com/bmatcuk/doublestar v1.3.1
 	github.com/containerd/containerd v1.3.4 // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible
```

Comment:
minor: that's not the latest version. is there a reason why we can't use that? cf https://github.com/evanphx/json-patch/tags

Reply:
Was the one we used before